src/validation.h

```diff
@@ -62,6 +62,8 @@ static const unsigned int DEFAULT_ANCESTOR_SIZE_LIMIT = 101;
 static const unsigned int DEFAULT_DESCENDANT_LIMIT = 25;
 /** Default for -limitdescendantsize, maximum kilobytes of in-mempool descendants */
 static const unsigned int DEFAULT_DESCENDANT_SIZE_LIMIT = 101;
+/** Default for -fullrbf, if the transaction opt-in replaceability is enforced */
+static const bool DEFAULT_TRANSACTION_REPLACEMENT = false;
```

Comment:
```suggestion
static constexpr bool DEFAULT_TRANSACTION_REPLACEMENT{false};
```

Reply:
I'd be more specific https://github.com/instagibbs/bitcoin/commit/27b1393438de56e4271d5d3de46a5de972d59fa6#diff-97c3a52bc5fad452d82670a7fd291800bae20c7bc35bb82686c2c0a4ea7b5b98R530

DEFAULT_FULL_RBF

Reply:
Went with `DEFAULT_MEMPOOL_FULL_RBF`.